main/lsp/LSPTypechecker.cc

```diff
@@ -31,6 +31,27 @@ void sendTypecheckInfo(const LSPConfiguration &config, const core::GlobalState &
             make_unique<NotificationMessage>("2.0", LSPMethod::SorbetTypecheckRunInfo, move(sorbetTypecheckInfo))));
     }
 }
+
+// In debug builds, used to assert that we have not accidentally taken the fast path after a change to the set of
```

Comment:
I read this as "used to assert (but no longer asserts)" not "this is used for the purpose of"
```suggestion
// In debug builds, asserts that we have not accidentally taken the fast path after a change to the set of
```